ansible/MANUAL_STEPS.md

```diff
@@ -1,5 +1,17 @@
 # Manual steps required to run ansible on machines
 
+## RHEL7-S390X
+1. V8 build-tools
+
+Copy `/home/iojs/build-tools/{gn,ninja}` from any other RHEL7-S390X host that
+has them to the newly provisioned one.
+
+If building from scratch, gn is built from  https://gn.googlesource.com/gn,
+and ninja is built from https://github.com/nodejs/node-v8.git
```

Comment:
Is this the right link for ninja?

Reply:
Hm, looks odd, I think it was what the remotes were setup on the build repos I found.